systemtest/src/test/java/io/strimzi/systemtest/migration/MigrationST.java

```diff
@@ -149,13 +206,90 @@ void testMigrationFromZkToKRaft() {
     @IsolatedTest
     void testRollbackDuringMigration() {
         TestStorage testStorage = new TestStorage(ResourceManager.getTestContext(), TestConstants.CO_NAMESPACE);
+        boolean zkDeleteClaim = true;
+        boolean deleteCoDuringProcess = false;
 
-        setupMigrationTestCase(testStorage);
-        doFirstPartOfMigration(testStorage);
+        setupMigrationTestCase(testStorage, zkDeleteClaim);
+        doFirstPartOfMigration(testStorage, deleteCoDuringProcess);
         doRollback(testStorage);
     }
 
-    private void setupMigrationTestCase(TestStorage testStorage) {
+    /**
+     * @description This testcase is focused on full circle of the migration process. The test starts with the migration,
+     * then, after the first phase, the migration is rolled back. After the rollback process is successfully completed, the test
```

Comment:
talking about rollback process completed could be misleading, because in the test you are moving from KRaftPostMigration to KRaftDualWriting but you are not moving back to ZooKeeper, which means that the rollback is really completed. Let's say you are stopping and resuming migration at half-way of a rollback process.

Reply:
yeah gonna change the comment here